app/assets/javascripts/admin/routes/admin-user-index.js.es6

```diff
@@ -19,6 +19,7 @@ export default Discourse.Route.extend({
     controller.setProperties({
       originalPrimaryGroupId: model.get("primary_group_id"),
       availableGroups: this._availableGroups,
+      customGroupIdsBuffer: null,
```

Comment:
I wonder, are these properties the only ones we need to reset?

Reply:
I will have a look at it and check if there are more things that need to be reset 👍

Reply:
@ZogStriP I tried a lot of things but all other user settings seem fine to me